internal/api/core/route.go

```diff
@@ -93,7 +93,7 @@ func NewAPIRoute(basePath string, apiHandler any) *APIRoute {
 
 	_, requestType, responseType, err := validateAPIHandler(fullPath, apiHandler)
 	if err != nil {
-		log.Error("Error while validating API handler.", err, log.NewAttr("endpoint", fullPath))
+		log.FatalWithCode(3, "Error while validating API handler.", err, log.NewAttr("endpoint", fullPath))
```

Comment:
I think your too deep to get to choose your exit status.
If you want, we can come up with a standard list.
Really, we would probably be following (sysexits.h)[https://man.freebsd.org/cgi/man.cgi?query=sysexits&sektion=3].